app/controllers/users/invitations_controller.rb

```diff
@@ -17,7 +17,7 @@ def create
           flash[:success] = "We invited '#{resource}' and sent them an email. You can update their settings as " \
                             'needed here.'
 
-          redirect_to edit_user_path(resource)
```

Comment:
When inviting a user we no longer will need to edit their status so we're going to return to the index page, and the newly added user will be on the top of the list in the sort.